docs/reference/buildx_bake.md

```diff
@@ -234,45 +408,42 @@ group "default" {
 target "webapp" {
     tags = ["docker.io/username/webapp:${TAG}"]
 }
-EOF
+```
 
+```console
 $ docker buildx bake --print webapp
 {
-   "target": {
-      "webapp": {
-         "context": ".",
-         "dockerfile": "Dockerfile",
-         "tags": [
-            "docker.io/username/webapp:latest"
-         ]
-      }
-   }
+  "target": {
+    "webapp": {
+      "context": ".",
+      "dockerfile": "Dockerfile",
+      "tags": [
+        "docker.io/username/webapp:latest"
+      ]
+    }
+  }
 }
+```
 
+```console
 $ TAG=$(git rev-parse --short HEAD) docker buildx bake --print webapp
```

Comment:
And then before this example, it could describe that the default value is overridden using the env-var.